cpp/open3d/t/geometry/TriangleMesh.h

```diff
@@ -487,6 +490,46 @@ class TriangleMesh : public Geometry, public DrawableGeometry {
     TriangleMesh SimplifyQuadricDecimation(double target_reduction,
                                            bool preserve_volume = true) const;
 
+    /// Computes the mesh that encompasses the union of the volumes of two
+    /// meshes.
+    /// Both meshes should be manifold.
+    ///
+    /// This function always uses the CPU device.
+    ///
+    /// \param mesh This is the second operand for the boolean operation.
+    /// \param tolerance Threshold which determines when point distances are
+    /// considered to be 0.
+    ///
+    /// \return The mesh describing the union volume.
+    TriangleMesh BooleanUnion(const TriangleMesh &mesh,
```

Comment:
LGTM.

Is it reasonable to call it `Union` instead of `BooleanUnion`? To put it in another way, will we ever have a union operation that is not "boolean"?

Reply:
Probably we won't have another union operation. I think it makes sense to add the prefix to make the function easier to find and it will also make all three operations appear next to each other.